Formula/zoxide.rb

```diff
@@ -23,6 +23,13 @@ def install
     fish_completion.install "contrib/completions/zoxide.fish"
   end
 
+  def caveats
+    <<~EOS
+      Before you start using zoxide, it must first be initialized on your shell.
+      See https://github.com/ajeetdsouza/zoxide#installation for instructions.
+    EOS
+  end
```

Comment:
We only carry `caveats` when they are Homebrew-specific (e.g. documenting paths where Homebrew installed things). This isn't Homebrew-specific, so this isn't something we add to `caveats`.

Reply:
I see, I'll close this PR then. Thanks!